static/bidder-info/smartadserver.yaml

```diff
@@ -0,0 +1,11 @@
+maintainer:
+  email: "support@smartadserver.com"
```

Comment:
This looks like it might more of a general support email. I just want to confirm that if we send an email about the code in this repo we will routed to the correct team at Smart.

Reply:
Yes I have double checked it, this will redirect to the relevant team :)